### PR TITLE
Added geokretymap.org

### PR DESCRIPTION
**Pull request by Rotzbua**

### CODE REVIEW

File: src/chrome/content/rules/geokretymap.org.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!--
+	Main page support https
+-->
+<ruleset name="geokretymap.org">
+	<target host="geokretymap.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/geokretymap.org.xml]: api.geokretymap.org

Mismatch on www.

**Comment by J0WI** [comment]
Thanks for adding this.
You may merge this into geokrety.org.xml or adding a comment on both files showing the relation.

**Comment by Rotzbua** [comment]
@J0WI rebased and updated. I think 2 separate files are better. I added a comment, that both files are related.

File: src/chrome/content/rules/geokrety.org.xml [hunk 2]
```diff
@@ -1,7 +1,12 @@
 <!--
 	Main pages support https
+	
+	Related page: geokretymap.org
```
Comment (J0WI) [inline src/chrome/content/rules/geokrety.org.xml]: Please use the filename

**Comment by Rotzbua** [comment]
@J0WI updated comment

**Comment by J0WI** [comment]
thanks!

**Pull request merged**